signer/src/transaction_coordinator.rs

```diff
@@ -535,6 +533,7 @@ where
 
         // Create a signal stream with the defined filter
         let signal_stream = self.context.as_signal_stream(presign_ack_filter);
+        let signature_threshold = self.context.config().signer.bootstrap_signatures_required;
```

Comment:
Can we add a comment that this may not be the "correct" threshold, but we don't really care since this is already a best effort check? eg, if we still need to rotate to the new threshold, we may check against the new (still unused) threshold. We should use something like the max of the thresholds of the DKG shares required to sign the inputs of the package, but assuming we will not do unreasonable changes to the threshold the config one should be fine.